rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -32,6 +32,8 @@ For more information on region availabilities, see xref:../rosa_architecture/ros
 
 * **Log linking is enabled by default.** Beginning with {product-title} 4.15, log linking is enabled by default. Log linking gives you access to the container logs for your pods.
 
+* **Cluster efficiency improvement** Now, you can perform specific tasks to make clusters more efficient. You can cordon, undoron, and drain a specific node. For more information, see xref:../rosa_cluster_admin/rosa_nodes/rosa-working-with-nodes[Working with nodes].
```

Comment:
I'm not sure this can be called 'Cluster efficiency improvement'. Perhaps just call this 'Node Management' or 'Node Administration'?

Reply:
"Node management improvements" would make sense here I think